aep/general/0202/aep.yaml

```diff
@@ -0,0 +1,7 @@
+---
+id: 202
+state: reviewing
+slug: fields
+created: 2023-01-22
+placement:
+  category: fields
```

Comment:
If this is going to stay a protobuf-specific AEP then should it be in the protocol buffers category (along with https://aep.dev/127)?

Reply:
We should add an OAS extension to this as well (see conversation below)

Reply:
For now, let's make this proto exclusive and we can revisit the OAS part later.